### ISSUE TITLE
i18n - Vite

### ISSUE THREAD

**Issue by tbordinat** [kenhyuwa/litepie-datepicker#12]
Using vite, i18n doesn't work.

```[vite] warning: 
/app/node_modules/.vite/litepie-datepicker.js
1997|        const locale = props.i18n;
1998|        nextTick(() => {
1999|          import(`dayjs/locale/${locale}.js`).then(() => {
|   |                 ^
2000|            import_dayjs.default.locale(locale);
2001|            let s, e;
The above dynamic import cannot be analyzed by vite.
See https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations for supported dynamic import formats. If this is intended to be left as-is, you can use the /* @vite-ignore */ comment inside the import() call to suppress this warning.```

**Comment by kenhyuwa** [comment]
#Fixed ([v1.0.13](https://github.com/kenhyuwa/litepie-datepicker/releases/tag/v1.0.13)).

**Comment by tbordinat** [comment]
Fixed, thank you so much !
